docs/team/radiankrisno.md

```diff
@@ -5,8 +5,72 @@ title: Radian Krisno's Project Portfolio Page
+    * Notable PRs: [PR #163](https://github.com/AY2122S1-CS2103T-T15-1/tp/pull/163)
+
+* **Ui Enhancement**: Implement UI component that displays the list of `Patient` stored in GoMedic in a tabular form, showing only some details.
+    * What it does: Creates a tabular view of the `Patient`s that users add into GoMedic.
+    * Credits: This command heavily uses [**JavaFX**](https://docs.oracle.com/javafx/2/). The format for this
+      tabular view of `Patient` closely follows the format and implementation done by my teammate, Simon,
+      who referenced this [blog](http://tutorials.jenkov.com/javafx/tableview.html) when creating his `TableView` for `Activity` .
+
+* **Overall Code Contribution**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&since=2021-09-17&timeframe=commit&mergegroup=AY2122S1-CS2103-F09-1%2Ftp%5Bmaster%5D&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=authorship&tabAuthor=radiankrisno&tabRepo=AY2122S1-CS2103T-T15-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
+
+* **Project management**:
+    * Opened issues that sought to improve to testing coverage , e.g. [Issue #164](https://github.com/AY2122S1-CS2103T-T15-1/tp/issues/164)
+    * Manage the [releases](https://github.com/AY2122S1-CS2103T-T15-1/tp/releases) of GoMedic
+    
+* **Notable Documentation**:
+    * User Guide:
+        * Wrote the [patient related features](https://ay2122s1-cs2103t-t15-1.github.io/tp/UserGuide.html#31-patients-related-features) section.
+        * Contribute in writing the [doctor related features](https://ay2122s1-cs2103t-t15-1.github.io/tp/UserGuide.html#32-doctors-related-features) section.
+    * Developer Guide:
+        * Revamped the [Logic Component](https://ay2122s1-cs2103t-t15-1.github.io/tp/DeveloperGuide.html#logic-component)
+          segment of the original AB3 DG. This included updating the Class Diagram, as well as adding more details about the implementation of the `Logic` class in general.
+        * Wrote the section about the implementation of the [view feature](https://ay2122s1-cs2103t-t15-1.github.io/tp/DeveloperGuide.html#view-feature).
+        * Wrote [Introduction](https://ay2122s1-cs2103t-t15-1.github.io/tp/DeveloperGuide.html#introduction)
+        * Wrote [Use Case 6, 7, and 8](https://ay2122s1-cs2103t-t15-1.github.io/tp/DeveloperGuide.html#use-cases)
+        * Wrote the manual testing instructions for [patients related features](https://ay2122s1-cs2103t-t15-1.github.io/tp/DeveloperGuide.html#viewing-a-patient)
+        * Wrote the manual testing instructions for [clear features](https://ay2122s1-cs2103t-t15-1.github.io/tp/DeveloperGuide.html#clearing-records-in-gomedic)
+        * Reformat manual testing [groupings](https://ay2122s1-cs2103t-t15-1.github.io/tp/DeveloperGuide.html#appendix-instructions-for-manual-testing)
```

Comment:
This one just combine with the above point of wrote manual testing, I think they are the same